app/services/billing_entities/update_service.rb

```diff
@@ -55,12 +55,7 @@ def call
 
         handle_eu_tax_management if params.key?(:eu_tax_management)
 
-        if License.premium? && billing.key?(:invoice_grace_period)
-          BillingEntities::UpdateInvoiceGracePeriodService.call(
-            billing_entity:,
-            grace_period: billing[:invoice_grace_period]
-          )
-        end
+        BillingEntities::UpdateInvoiceIssuingDateSettingsService.call(billing_entity:, params:)
```

Comment:
Not really important but maybe we could pass only the `params[:billing_configuration]`